File: task-03/sub-task 2/Java/code.java
import java.nio.file.Files;
import java.nio.file.Paths;
import java.io.IOException;

public class Main {
    public static void main(String[] args) {
        try {
            String content = Files.readString(Paths.get("/home/abinav/git/amfoss-tasks/task-03/sub-task 2/input.txt"));
            Files.writeString(Paths.get("/home/abinav/git/amfoss-tasks/task-03/sub-task 2/Java/output.txt"), content);
        } catch (IOException e) {
            e.printStackTrace();
        }
    }
}

File: task-03/sub-task 2/Rust/code.rs
use std::fs;

fn main() {
    let content = fs::read_to_string("/home/abinav/git/amfoss-tasks/task-03/sub-task 2/input.txt")
        .expect("Failed to read input.txt");
    fs::write("/home/abinav/git/amfoss-tasks/task-03/sub-task 2/Rust/output.txt", content)
        .expect("Failed to write to output.txt");
}
